dashboard/app/controllers/registrations_controller.rb

```diff
@@ -163,13 +163,14 @@ def create
           error_message: "retry ##{retries} failed with exception: #{exception}"
         )
       end
-      super
-    end
 
-    if params[:new_sign_up].present?
-      session[:user_return_to] ||= params[:user_return_to]
-      @user = Services::PartialRegistration::UserBuilder.call(request: request)
-      sign_in @user
+      if params[:new_sign_up].present?
+        session[:user_return_to] ||= params[:user_return_to]
+        @user = Services::PartialRegistration::UserBuilder.call(request: request)
+        sign_in @user
+      else
+        super
+      end
```

Comment:
Moves the new signup `UserBuilder` call inside the `Retryable` block to mirror the old flow's behavior

Reply:
And calling super isn't needed for the new flow?

Reply:
Yep!